coderd/notifications/metrics_test.go

```diff
@@ -43,7 +44,7 @@ func TestMetrics(t *testing.T) {
 
 	reg := prometheus.NewRegistry()
 	metrics := notifications.NewMetrics(reg)
-	template := notifications.TemplateWorkspaceDeleted
+	tmplate := notifications.TemplateWorkspaceDeleted
```

Comment:
typo: template?

Reply:
renamed to tmpl due to package import conflict.